aspnetcore/fundamentals/logging/index.md

```diff
@@ -233,6 +233,8 @@ Calling `CreateLogger` with a fixed name can be useful when used in multiple met
 
 `ILogger<T>` is equivalent to calling `CreateLogger` with the fully qualified type name of `T`.
 
+***NOTE:*** [LoggerFactory.Create](/dotnet/api/microsoft.extensions.logging.loggerfactory.create) Creates a new instance of <xref:Microsoft.Extensions.Logging.ILoggerFactory> that's configured using the provided `configure` delegate. The new instance of `ILoggerFactory` creates a new logging pipeline, different from the default logging pipeline. The new logging pipeline is not configured by the `Logging` section of `appsettings.json` or `appsettings.{ENVIRONMENT}.json`. For more information, see [Logging configuration](xref:fundamentals/logging/index#configure-logging).
```

Comment:
The text of the note seems quite accurate to me, but the note itself feels disconnected from the text around it. Assuming that we make the other changes in the snippets then this docs page will have no references to using LoggerFactory.Create and I think the note can be omitted. I am guessing a lot of the confusion was occurring because some sample snippets were using LoggerFactory.Create and people would copy those snippets.

Reply:
> Assuming that we make the other changes in the snippets then this docs page will have no references to using LoggerFactory.Create and I think the note can be omitted.

+1

Reply:
(Very impressed with ChatGPT's notes, btw!)

Reply:
@Rick-Anderson I suggest to remove this note. Its no longer needed, and having it there is probably confusing without any context.